remo/profiles/templates/profiles_view.jinja

```diff
@@ -348,15 +348,19 @@ Mozilla Reps - Profile of {{ user_profile.display_name }}
             <span class="label radius">
```

Comment:
I think it would make sense to not show the labels either, as when not logged in, we won't have any value at all. Do you agree?

<img width="407" alt="screen shot 2018-06-15 at 14 55 09" src="https://user-images.githubusercontent.com/330324/41491154-51764c44-70ac-11e8-9f7b-0542eb775fe2.png">